padle/menu/globalstats.py

```diff
@@ -0,0 +1,73 @@
+from discord import Message
+from discordmenu.embed.menu import EmbedMenu
+from discordmenu.embed.wrapper import EmbedWrapper
+from padle.view.globalstats_view import GlobalStatsView, GlobalStatsViewState
+from tsutils.menu.components.panes import MenuPanes
+from typing import Optional
+
+
+class ScrollEmojis:
+    prev_page = '\N{BLACK LEFT-POINTING DOUBLE TRIANGLE}'
+    next_page = '\N{BLACK RIGHT-POINTING DOUBLE TRIANGLE}'
+
+
+class GlobalStatsMenu:
+    MENU_TYPE = 'GlobalStatsMenu'
+
+    view_types = {
+        GlobalStatsViewState.VIEW_STATE_TYPE: GlobalStatsView
+    }
+
+    @staticmethod
+    def menu(initial_control=None):
+        if initial_control is None:
+            initial_control = GlobalStatsMenu.control
+        embed = EmbedMenu(GlobalStatsMenuPanes.transitions(), initial_control)
+        return embed
+
+    @classmethod
+    async def respond_with_left(cls, message: Optional[Message], ims, **data):
+        if ims['current_day'] < ims['num_days']:
```

Comment:
Now that you know what you're doing with menu2, check out padinfo id menu and see if you can make these a staticmethod on the view state
```
@staticmethod
def decrement_day
```

and same thing for increment_day

it's really best not to directly modify the ims inside of the menu code (e.g. if you decided to refactor the parameter name "current_day" you would want all necessary modifications to take place in the view file)